sdk/identity/azure-identity/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.identity {
+    // FIXME this is unfortunate - java.desktop is used to open the users browser
+    requires java.desktop;
```

Comment:
Can you file a tech debt on this please?

Reply:
Will create an issue.